wgpu/src/util/init.rs

```diff
@@ -0,0 +1,91 @@
+use wgt::{BackendBit, PowerPreference, RequestAdapterOptions};
+
+use crate::{Adapter, Instance};
+
+/// Get a set of backend bits from the environment variable WGPU_BACKEND.
+pub fn backend_bits_from_env() -> Option<BackendBit> {
+    Some(
+        match std::env::var("WGPU_BACKEND")
+            .as_deref()
+            .map(str::to_lowercase)
+            .as_deref()
+        {
+            Ok("vulkan") => BackendBit::VULKAN,
+            Ok("dx12") => BackendBit::DX12,
+            Ok("dx11") => BackendBit::DX11,
+            Ok("metal") => BackendBit::METAL,
+            Ok("gl") => BackendBit::GL,
+            Ok("webgpu") => BackendBit::BROWSER_WEBGPU,
+            _ => None?,
```

Comment:
is clippy not yelling at you for this sin? it should :)

Reply:
But think about all 6 characters it saved 😆